docs/CONTRIBUTING.md

```diff
@@ -249,3 +250,48 @@ These contribution guidelines were inspired by and are based on Atom’s
 contribution guidelines. They were modified for the purposes of this repository.
 https://github.com/atom/atom/blob/master/CONTRIBUTING.md - Copyright (c)
 2011-2017 GitHub Inc. (MIT)
+
+### JSDoc Styleguide
```

Comment:
The title should make clear that it's for types, in case people are unfamiliar with JSDoc.

Reply:
Gonna change it to `Code documentation Styleguide` to make it clear.